package.json

```diff
@@ -110,6 +110,20 @@
           "import/no-commonjs": 2
         }
       },
+      {
+        "files": [
+          "**/packages/**/*.ts",
+          "**/packages/**/*.tsx"
+        ],
+        "rules": {
+          "prettier/prettier": [
+            "error",
+            {
+              "parser": "typescript"
```

Comment:
since we are going to use Babel for parsing TS maybe it would be better to use `babel-ts` here instead?

Reply:
That sounds fine! We'd need to upgrade `prettier` in order to use it though. Is that fine to include?

Reply:
ye, go for it

Reply:
actually - maybe leave that off for a followup PR